hooks/post-up

```diff
@@ -6,4 +6,15 @@ if [ ! -e $HOME/.vim/autoload/plug.vim ]; then
   curl -fLo $HOME/.vim/autoload/plug.vim --create-dirs \
       https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
 fi
+
 vim -u $HOME/.vimrc.bundles +PlugInstall +PlugClean! +qa
+
+if grep --quiet --silent 'path_helper' /etc/zprofile; then
```

Comment:
According to `man grep` on OS X, `--quiet` and `--silent` do the same thing:

```
 -q, --quiet, --silent
             Quiet mode: suppress normal output.  grep will only search a file until a match has been found, making searches potentially less expensive.
```

Do you mean `--quiet --no-messages`?

```
     -s, --no-messages
             Silent mode.  Nonexistent and unreadable files are ignored (i.e. their error messages are suppressed).
```

Is there a difference between OS X, *BSD, and Linux grep here? @thoughtbot/shell